content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -251,7 +251,7 @@ If your configuration is not using the latest version it is **recommended** that
 the [kubeadm config migrate](/docs/reference/setup-tools/kubeadm/kubeadm-config/) command.
 
 For more information on the fields and usage of the configuration you can navigate to our
-[API reference page](/docs/reference/config-api/kubeadm-config.v1beta3/).
+[API reference page](/docs/reference/config-api/kubeadm-config.v1beta4/).
```

Comment:
We are still using v1beta3 now. V1beta4 is still in early age in development process in v1.29.
/cc @chendave @neolit123

Reply:
Should we revert the changes made in the [English version](https://github.com/kubernetes/website/pull/42642/files#diff-f50518e6a9a01e98b5d7069bb273c95f2d2dbe919a97f434c15fda9e75aaa523) of `main` branch?

Reply:
user shouldn't be able to init a cluster by kubeadm with v1beta4 as the API is not registered, might be some note there to avoid some confusion.

Reply:
https://github.com/kubernetes/website/pull/42642/files#diff-f50518e6a9a01e98b5d7069bb273c95f2d2dbe919a97f434c15fda9e75aaa523L138-L298

- only part of that PR: content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

Reply:
Ok, will remove this file from this PR and sync it after en upstream is changed.